dub.json

```diff
@@ -10,7 +10,7 @@
 	"targetPath": "bin",
 	"dependencies": {
 		"derelict-pq": "~>3.0.0-alpha.2",
-		"vibe-d:data": "~>0.7.27-rc.2"
+		"vibe-d:data": ">=0.7.27 <0.9.0"
```

Comment:
yes, but we have `<` there meaning we reject it (who knows what changes might be in that release)